src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -175,6 +175,35 @@ sequences using different contrast enhanced images. The label is the name of the
 contrast agent. The key `ContrastBolusIngredient` MAY be also be added in the
 JSON file, with the same label.
 
+#### The `part` entity
+
+This entity is used to indicate which component of the complex representation of the MRI signal is represented in voxel data.
+The `part-<label>` key/value pair is associated with the DICOM tag `0008,9208`.
+Allowed label values for this entity are `phase`, `mag`, `real` and `imag`, which are typically used in `mag/phase` or `real/imag` pairs.
+For example:
+
+```Text
+sub-01_part-mag_T1w.nii.gz
+sub-01_part-mag_T1w.json
+sub-01_part-phase_T1w.nii.gz
+sub-01_part-phase_T1w.json
+```
+
+Phase images MAY be in radians or in arbitrary units.
+The sidecar JSON file MUST include the units of the `phase` image.
+The possible options are: `radians` or `a.u.` for grayscale images.
```

Comment:
Related to #580, do we want "a.u." or "arbU"?

Reply:
Agreed. I'd prefer to remove "grayscale", although I believe it's used fairly frequently in #508.